README.md

```diff
@@ -1 +1,55 @@
-# Awesome-Books-2
\ No newline at end of file
+# Awesome-Books-2
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# AwesomeBooks_ver2.0
+
+> A Modular version of Awesome Books. In this version of Awesome book, js codes are organised in modules.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Getting Started
+
+**Clone the repo into your local machine.**
+
+Source Code
+
+[source code](https://github.com/ShaheerCH/Awesome-Books-2.git)
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
- [ ] [optional] It would have been nice adding a line on how to go about cloning the project, it's a good practice since not everyone visiting your repo has knowledge about cloning.